docs/gravitino-server-config.md

```diff
@@ -138,6 +138,23 @@ The plugin provides several operational modes for how to process event, supporti
 
 For more details, please refer to the definition of the plugin.
 
+### Audit log configuration
+
+The audit log framework defines how metadata changes are logged to various storages. The formatter defines an interface that transforms different `Event` types into a unified `AuditLog`.
+The writer defines an interface for writing to different storage types.
```

Comment:
The writer defines an interface for writing to different storage types. -> The writer defines an interface to writing `AuditLog` to different storages. ?

Reply:
please add `Gravitino provides a default implement to log basic audit information to a file, you could extend the audit system by implementation corresponding interfaces.`

Reply:
Could you combine the two line into one line?